.stylelintrc.json

```diff
@@ -1,11 +1,12 @@
 {
-	"extends": "stylelint-config-wordpress",
+	"extends": "@wordpress/stylelint-config",
```

Comment:
This was a breaking change needing fixed from the `@wordpress/scripts` update because `@wordpress/scripts` no longer bundles `stylint-config-wordpress`.